### PR TITLE
Breaking: Form styles

### PR DESCRIPTION
**Pull request by omgaz**
screen shot 2016-02-09 at 14 23 45

### CODE REVIEW

**Comment by camjc** [comment]
I guess this should be BREAKING: not New

File: src/styles/bootstrapOverrides/Form.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  &:last-child {
+    padding-right: 0;
+  }
+}
+
+.help-block {
+  color: $color-text-placeholder;
+  font-weight: $font-weight-medium;
+  margin-top: 8px;
```
Comment (camjc) [inline src/styles/bootstrapOverrides/Form.scss]: :crystal_ball: Magic number?

Comment (omgaz) [inline src/styles/bootstrapOverrides/Form.scss]: It looks right

File: src/styles/variable.scss [hunk 2]
```diff
@@ -115,19 +121,19 @@ $pagination-active-border: $color-border-lighter;
...
 
-$state-warning-text: $color-text-inverse;
+$state-warning-text: $color-warning;
 $state-warning-bg: $color-warning;
 $state-warning-border: $color-warning-selected;
 
-$state-danger-text: $color-text-inverse;
+$state-danger-text: $color-negative;
 $state-danger-bg: $color-negative;
 $state-danger-border: $color-negative-selected;
```
Comment (antklim) [inline src/styles/variable.scss]: We need colour `danger` to support bootstrap-ui

Comment (camjc) [inline src/styles/variable.scss]: The names on the right are our own names, the names on the left trickle into bootstrap.
All the css for bootstrap-ui comes from bootstrap anyway.
This code is good.

**Comment by camjc** [comment]
Merging!

**Pull request merged**